### ISSUE TITLE
Make prompt in echo command the same color as normal prompt

### ISSUE THREAD

**Issue by jcubic** [jasonbio/reddit-shell#10]
If you type something the current prompt get echoed but it's gray not blue. You can fix that by modifiing set_prompt call:

```javascript
term.set_prompt('[[b;#2FD0C3;]' + auth_user+'@reddit:~$ ' + ']');
```

then when you type enter you'll get the same style (and css will not be needed).

**Comment by jasonbio** [comment]
I believe this is fixed by the latest commit which updates jQuery Terminal to the latest version. Please feel free to verify that this resolves your issue

**Comment by jcubic** [comment]
You need to update css as well since now it show textarea that was used for resize event in older Chrome and other browsers that don't support resizeObserver.

**Comment by jasonbio** [comment]
Thanks for letting me know, this is addressed in 0fc381d
